core/platform/models.py

```diff
@@ -249,7 +250,7 @@ def _get(cls):
         Returns:
             class: The corresponding platform-specific interface class.
         """
-        return cls._PLATFORM_MAPPING.get(feconf.PLATFORM)
+        return cls._PLATFORM_MAPPING.get(Registry.GAE_PLATFORM)
```

Comment:
Semantically, this method is supposed to be platform-independent, so better to define CURRENT_PLATFORM as a module-level constant in this file, and refer to that instead.

Reply:
Done.